scripts/start-dev-infra.sh

```
@@ -5,6 +5,15 @@ set -euo pipefail
 ROOT_DIR="$(cd -- "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 cd "$ROOT_DIR"
 
+# Load environment vars from .env.local from repo root
+ENV_FILE="$ROOT_DIR/.env.local"
+if [ -f "$ENV_FILE" ]; then
+  echo "💉 Loading $ENV_FILE"
+  set -a
+  . "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)